src/nvidia-cuda/install.sh

```diff
@@ -17,6 +17,11 @@ if [ "$(id -u)" -ne 0 ]; then
     exit 1
 fi
 
+if [[ $(uname -m) = "aarch64" ]]; then
+    echo -e "NVIDIA® CUDA Toolkit no longer supports development or running applications on machines with the 'arm64' architecture. Please consider an alternative or check for updates or support for your architecture."
+    exit 1
```

Comment:
Doesn't https://developer.nvidia.com/nvidia-cuda-toolkit-11_7_0-developer-tools-mac-hosts refer to NVIDIA toolkit no longer supporting MacOS? 

Hence, instead of failing the entire Feature, why don't we update 👇 instead?

https://github.com/devcontainers/features/blob/9f152b5725273fef7d863d81e29c7e03d5263f14/src/nvidia-cuda/install.sh#L95-L98